story-site/static/styles.css

```
@@ -0,0 +1,468 @@
+  --ash-deep: #141716;
+  --line: rgba(61, 70, 66, 0.24);
+  --line-dark: rgba(233, 228, 213, 0.17);
+  --copper: #b06f49;
+  --copper-bright: #d89c70;
+  --verdigris: #4b756d;
+  --verdigris-bright: #7ba59b;
+  --warning: #8f4e38;
+  --radius: 0.35rem;
+  --shadow: 0 28px 90px rgba(0, 0, 0, 0.22);
+  --measure: 72ch;
+  font-family: Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)